diff --git a/preflight-cli/src/main/java/dev/starsector/preflight/cli/PrepareCommand.java b/preflight-cli/src/main/java/dev/starsector/preflight/cli/PrepareCommand.java
index 87c045f..0000000 100644
--- a/preflight-cli/src/main/java/dev/starsector/preflight/cli/PrepareCommand.java
+++ b/preflight-cli/src/main/java/dev/starsector/preflight/cli/PrepareCommand.java
@@ -487,7 +487,7 @@ final class PrepareCommand {
         }
 
         boolean successful() {
-            return "SUCCESS".equals(status);
+            return !"FAILED".equals(status);
         }
 
         Map<String, Object> toMap() {
